pyQuARC/code/custom_validator.py

```diff
@@ -98,29 +98,23 @@ def one_item_presence_check(*field_values):
 
         return {"valid": validity, "value": value}
 
-    @staticmethod
-    @if_arg
-    def opendap_url_location_check(field_value):
-        # The field shouldn't have a opendap url
-        return {"valid": "opendap" not in field_value.lower(), "value": field_value}
-
     @staticmethod
     @if_arg
     def user_services_check(first_name, middle_name, last_name):
         return {
-            "valid": not (
-                first_name.lower() == "user"
-                and last_name.lower() == "services"
-                and (not middle_name or (middle_name.lower() == "null"))
+            "valid": (
+                first_name.lower() != 'user' or
+                last_name.lower() != 'services' or 
+                (middle_name and (middle_name.lower() != 'null'))
             ),
             "value": f"{first_name} {middle_name} {last_name}",
         }
 
     @staticmethod
     def doi_missing_reason_explanation(explanation, missing_reason, doi):
         return {
-            "valid": not ((not doi) and (missing_reason) and (not explanation)),
-            "value": explanation,
+            "valid": doi or not missing_reason or explanation,
```

Comment:
what is this check?

Reply:
@xhagrg : https://github.com/NASA-IMPACT/pyQuARC/issues/73